### ISSUE TITLE
Support Single fetch in Remix

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/remix-og-image#4]
In the future versions of Remix, [Single Fetch](https://remix.run/docs/en/main/guides/single-fetch#resource-routes) will be a thing. It will allow this plugin to fetch resource data from _all_ OG image routes _at once_:

```
GET /a/b/c.data?_routes=routes/a,routes/b
```

This is an experimental feature behind `future: { unstable_singleFetch }` so I cannot rely on it by default.

**Comment by kettanaito** [comment]
Need to handle this because the plugin breaks if you are using experimental Single Fetch: Remix throws on `_data` requests now. I don't see an alternative to `_data` requests with SF. `.data?_routes` does _not_ produce the same result. It returns a bunch of internal JSON describing the routes. That's not the goal. The goal is to trigger a UI route to act as a resource route in a particular scenario.

**Comment by kettanaito** [comment]
Raised this with the Remix team: https://github.com/remix-run/remix/discussions/7640#discussioncomment-10374933

**Comment by kettanaito** [comment]
## Discovery

Okay, so this turns out to be not an internal json but something called _turbo stream_:

```json
[{"_1":2,"_13":14},"root",{"_3":4},"data",{"_5":6,"_7":8,"_9":10,"_11":12},"colorScheme","system","host","localhost:3000","isProductionHost",false,"noIndex",true,"routes/og.blog.$slug",{"_3":15},{"_16":17},"hello","world"]
```

And it indeed contains the json payload returned from the `loader`. 

I need to use the `turbo-stream` package to decode it if the user is using experimental Single fetch with Remix.

Also, to trigger a resource route, `.data?_route` is enough, no need to include the route id.

**Comment by kettanaito** [comment]
## Released: v0.3.0 🎉

This has been released in v0.3.0!

- 📄 [**Release notes**](https://github.com/kettanaito/remix-og-image/releases/tag/v0.3.0)
- 📦 [npm package](https://www.npmjs.com/package/remix-og-image/v/0.3.0)

Make sure to always update to the latest version (`npm i remix-og-image@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
